lib/Controllers/update.js

```diff
@@ -47,10 +28,45 @@ Update.prototype.write = function(req, res, context) {
       context.attributes[a] = req.params[a];
   });
 
+  // Update associated data
+  var self = this,
+      associationPrimKeys = [];
+  if (this.include && this.include.length) {
+    this.include.forEach(function(i) {
+      var instanceModel = i.model || i,
```

Comment:
Replace https://github.com/dchester/epilogue/blob/168332280ef0bd5d5660c59552e4f4a1075a285b/lib/Resource.js#L21 by 
```
  this.include = options.include.map(function(include) {
    if (!include.model) {
      include = { model: include };
    }
    return include;
  });
```
 ?

Tested and it work. I can add this on this MR. @mbroadst